9.2/root/usr/share/container-scripts/postgresql/README.md

```diff
@@ -33,6 +33,9 @@ You can also set the following mount points by passing the `-v /host:/container`
 | :----------------------- | ------------------------------------- |
 |  `/var/lib/pgsql/data`   | PostgreSQL database cluster directory |
 
+
+At location `${CONTAINER_SCRIPTS_PATH}/post-init` should be located script which defines all post initialization tasks. By default there is no script. You can either use this image as base image. In this case you should do `COPY path-to-your-script ${CONTAINER_SCRIPTS_PATH}/post-init`. Other way is to mount just single file from host machine. This way you must use -v flag `docker -v path-to-your-script:${CONTAINER_SCRIPTS_PATH}/post-init`. You can check more about mouting single file [at docker wiki](https://docs.docker.com/engine/userguide/containers/dockervolumes/#mount-a-host-file-as-a-data-volume).
```

Comment:
I doubt the ${CONTAINER_SCRIPTS_PATH} will work in the case of `docker run` (missing the `run` part in the command above).